locales/en/messages.json

```diff
@@ -2918,7 +2918,7 @@
         "message": "Throttle Boost"
     },
     "pidTuningThrottleBoostHelp": {
-        "message": "This feature allows throttle to be temporarily boosted to the up to down-side to increase acceleration torque to the motors, providing a much faster throttle response."
+        "message": "This feature allows throttle to be temporarily boosted on quick stick movements, winch increases acceleration torque to the motors, providing a much faster throttle response"
```

Comment:
You are including a new one: winch 😋

Reply:
haha oops, I guess I've spelled that wrong my whole life :P fixed!